node/src/db_config/config_dao_null.rs

```diff
@@ -283,4 +295,41 @@ mod tests {
 
         assert_eq!(value_pairs, expected_pairs);
     }
+
+    #[test]
+    fn set_works_simple() {
+        let subject = ConfigDaoNull::default();
+
+        subject.set("param1", Some("value1".to_string())).unwrap();
+        subject.set("schema_version", None).unwrap();
+        subject
+            .set("schema_version", Some("456".to_string()))
+            .unwrap();
+
+        let subject_data_sorted = subject.data.iter().sorted().collect::<Vec<(_, _)>>();
```

Comment:
Let's consider if the `set()` was modifying the above mentioned changes, the assert would still succeed because we just took the same reading twice.

It would be a stronger assertion if we move the `subject_data_sorted` before the act.